src/cloud/apis/reporting.ts

```diff
@@ -21,20 +22,28 @@ export interface Points {
 }
 
 export const reportPoints = (points: Points) => {
-  if (!isFlagEnabled('appMetrics')) {
-    return
+  if (isFlagEnabled('rudderstackReporting')) {
+    try {
+      points.points.forEach(point => {
+        track(point.measurement, point.fields, point.tags, () => {})
+      })
+    } catch (e) {
+      // don't want reporting errors to effect user experience
```

Comment:
wouldn't it be helpful for us to know if the service falls over? Maybe a honeybadger alert or something?

Reply:
I followed the pattern below for app-metrics.  But I agree that sounds better and I'll look into adding this.  (Do you think it's worth adding for the appMetrics call below as well at the same time?)

Reply:
Truthfully I have no idea what appMetrics is for so i can't say

Reply:
Added a Honeybadger report on failure here - if it gets noisy or problematic we can remove it later but that's better than not knowing.  I left the appMetrics error being swallowed.